asreview/models/cli.py

```diff
@@ -21,17 +21,13 @@ def _format_algorithm(values, name, description):
     result = []
 
     for x in values:
-        if not getattr(x, "label", None):
-            if hasattr(x, "dist") and x.dist:
-                if x.dist.name != "asreview":
-                    x.label = f"[{x.dist.name} {x.dist.version}]"
-
-        if hasattr(x, "label"):
-            result.append(
-                " " * 22 + f"{x.name}" + " " * (16 - len(x.name)) + f"{x.label}"
-            )
-        else:
-            result.append(" " * 22 + f"{x.name}")
+        padding = " " * 22 + x.name
+        if hasattr(x, "dist") and x.dist:
+            if x.dist.name != "asreview":
```

Comment:
```suggestion
        if hasattr(x, "dist") and x.dist.name != "asreview":
```

Is this the same?

Reply:
I think when you do `x.dist.name` when `x.dist` is None, it will throw an error.

Reply:
If you combined these conditions into one, and x.dist is None or falsy (but still has a .dist attribute), this could raise an AttributeError or a NoneType error.